docs/reference/search/aggregations/bucket/histogram-aggregation.asciidoc

```diff
@@ -326,6 +326,15 @@ Here is an example of what the response could look like:
 }
 --------------------------------------------------
 
+==== Offset
+
+By default the bucket keys start with 0 and then continue in even spaced steps of `interval`, e.g. if the interval is 10 the first buckets
+(assuming there is data inside them) will be [0 - 9), [10-19), [20-29). The bucket boundaries can be shifted by using the `offset` option.
```

Comment:
Do you mean `[0 - 9]` or `[0 - 10)` instead of `[0 - 9)` (I'm assuming '[' means including while '(' means excluding?)